components/Header/Header.js

```diff
@@ -1,9 +1,39 @@
 import styled from "styled-components";
+import { useRouter } from "next/router";
 
 const StyledHeadline = styled.h1`
   text-align: center;
 `;
 
+const StyledHeader = styled.header`
+  position: relative;
+`;
+
+const StyledBackButton = styled.button`
+  position: absolute;
+  top: 6%;
+  left: 5%;
+  background-color: white;
+  border: none;
+  font-size: 1.5rem;
+  &:hover {
+    cursor: pointer;
+  }
+`;
+
 export default function Header() {
-  return <StyledHeadline>Shelfie</StyledHeadline>;
+  const router = useRouter();
+
+  return (
+    <StyledHeader>
+      {router.pathname !== "/" && (
+        <StyledBackButton onClick={() => router.back()}>
+          <span role="image" aria-label="go back to previous site button">
+            ⬅️
```

Comment:
A super petty hint to formulate the aria label more explicitly. 🤓

```suggestion
          <span role="image" aria-label="Arrow emoji indicating the return to the previous page">
            ⬅️
```